### PR TITLE
[RA2 Ch4]: Nile proofreading changes

### LINKED ISSUE #3269: [RA2 Ch4] Nile Proofreading

### PR DESCRIPTION
**Pull request by CsatariGergely**
Fixes #3269

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 1]
```diff
@@ -4,16 +4,16 @@ Component Level Architecture
 Introduction
 ------------
 
-This chapter describes in detail the Kubernetes Reference Architecture in terms
-of the functional capabilities and how they relate to the Reference Model
+This chapter describes in detail the Reference Architecture (RA2) for Kubernetes based cloud infrastructure in terms
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:7]:
```suggestion
This chapter describes in detail the Reference Architecture (RA2) for Kubernetes-based cloud infrastructure in terms
```

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 2]
```diff
@@ -100,22 +100,37 @@ the following specifications:
    * - ra2.ch.010
      - Local Filesystem Storage Quantity
      - The Kubernetes Nodes must be equipped with local filesystem capacity of at least 320GB for unpacking and
-       executing containers. Note, extra should be provisioned to cater for any overhead required by the Operating
-       System and any required OS processes such as the container runtime, Kubernetes agents, etc.
+       executing containers.
+
+       .. note::
+
+        Note, extra should be provisioned to cater for any overhead required by the Operating
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:107]:
```suggestion
        Note, extra filesystem storage should be provisioned to cater for any overhead required by the Operating
```

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 3]
```diff
@@ -100,22 +100,37 @@ the following specifications:
...
    * - ra2.ch.011
      - Virtual Node CPU Quantity
-     - If using VMs, the Kubernetes Nodes must be equipped with at least 16 vCPUs. Note, extra should be provisioned to
-       cater for any overhead required by the Operating System and any required OS processes such as the container
-       runtime, Kubernetes agents, etc.
+     - If using VMs, the Kubernetes Nodes must be equipped with at least 16 vCPUs. 
+
+       .. note::
+
+        Note, extra should be provisioned to
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:118]:
```suggestion
        Note, extra CPU capacity should be provisioned to
```

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 4]
```diff
@@ -100,22 +100,37 @@ the following specifications:
...
    * - ra2.ch.012
      - Kubernetes Node RAM Quantity
-     - The Kubernetes Nodes must be equipped with at least 32GB of RAM. Note, extra should be provisioned to cater for
-       any overhead required by the Operating System and any required OS processes such as the container runtime,
-       Kubernetes agents, etc.
+     - The Kubernetes Nodes must be equipped with at least 32GB of RAM.
+     
+       .. note::
+
+        Note, extra should be provisioned to cater for
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:130]:
```suggestion
        Note, extra RAM capacity should be provisioned to cater for
```

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 5]
```diff
@@ -283,24 +298,34 @@ the following specifications:
...
-       must be enabled and configured on the kubelet (note, TopologyManager is enabled by default in Kubernetes v1.18
-       and later, with CPUManager enabled by default in Kubernetes v1.10 and later).
+       must be enabled and configured on the kubelet.
        --feature-gates="..., TopologyManager=true,CPUManager=true" --topology-manager-policy=single-numa-node
        --cpu-manager-policy=static
+
+       .. note::
+
+          Note, TopologyManager is enabled by default in Kubernetes v1.18 and later, with CPUManager enabled by default
+          in Kubernetes v1.10 and later
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:314]:
```suggestion
          in Kubernetes v1.10 and later.
```

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 6]
```diff
@@ -283,24 +298,34 @@ the following specifications:
...
    * - ra2.k8s.007
      - DevicePlugins Feature Gate
-     - When hosting workloads matching the High Performance profile, the DevicePlugins feature gate must be enabled
-       (note, this is enabled by default in Kubernetes v1.10 or later). --feature-gates="...,DevicePlugins=true,..."
+     - When hosting workloads matching the High Performance profile, the DevicePlugins feature gate must be enabled. 
+       --feature-gates="...,DevicePlugins=true,..."
+
+       .. note::
+
+        Note, this is enabled by default in Kubernetes v1.10 or later
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:327]:
```suggestion
        Note, this is enabled by default in Kubernetes v1.10 or later.
```

**Review by tomkivlin — COMMENTED** [review]
Some minor changes:
- Clarity to some notes
- Consistent use of full-stop

No need to block the PR merge though.

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 7]
```diff
@@ -649,7 +679,7 @@ Architecture they must be implemented as per the following specifications:
    * - ra2.pkg.001
      - API-based package management
      - A package manager must use the Kubernetes APIs to manage application artifacts. Cluster-side components such as
-       Tiller are not supported.
+       Tiller must not be required.
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:682]: should it just be "not required"
```suggestion
       Tiller are not required.
```

Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter04.rst:682]: I would prefer to keep it like this, as "are not required" means that it is optional, be we would like to forbid its usage.

**Review by rgstori — APPROVED** [review]

**Review by steelescotr — APPROVED** [review]

**Review by tomkivlin — APPROVED** [review]

**Pull request merged**